### DISCUSSION TITLE
Como se podría mejorar las respuestas de los errores mostrados en el automatic log?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#143] [3. Ideas & Feature Requests] [answered]
Hola

Quisiera saber si podría existir (mas adelante o a mediano plazo), una forma que te puede entregar más información de los errores que muestra en el automatic log, debido a que me he encontrado con 2 tipos de errores que lso dejo en detalle

![image](https://user-images.githubusercontent.com/61419471/117019991-fb523100-acc3-11eb-8c08-93e5ec87b698.png)
 **Comentario:** Se que al tener ↑ ↑ ┴──────────────────────────────────────────────────────────────────────────────────────────────────────┘ significa que el código no es compatible al cotlang y se deba cambiar. Sin embargo no me indica como debería verse.

![image](https://user-images.githubusercontent.com/61419471/117020227-3fddcc80-acc4-11eb-91f0-b704364af7aa.png)
**Comentario:** Arroja un error 500, pero no me puede decir en qué parte sucede :(.

Eso gracias

**Comment by acedward** [comment, marked as answer]
En COTLang debes escapar los strings, es la forma más fácil de que funcionen
debes poner triple ticks en los strings litearles y no paréntesis.  

![image](https://user-images.githubusercontent.com/3951895/117021776-a3b4c500-acc5-11eb-9757-918705c886ba.png)

**Reply by NicolasPLG** [reply]
Entiendo @acedward gracias por la aclaración :)
